cosmovisor/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/cosmos/cosmos-sdk/cosmovisor
 go 1.18
 
 require (
-	github.com/cosmos/cosmos-sdk v0.46.1
+	github.com/cosmos/cosmos-sdk v0.46.0-beta2.0.20220909113810-4882f933b1a1
```

Comment:
why pin against this and not a semver?

Reply:
Because the migration to cosmos/gogoproto isn't released yet.
As we import the sdk for the upgrade plan only, we can just use devel until we release 0.47

Reply:
got it 👍